reducers/views/rhs_stash.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {ViewsState} from 'types/store/views';
+import {ActionTypes} from 'utils/constants';
+
+export default function rhsStash(state: ViewsState['rhsStash'] = null, action: GenericAction): ViewsState['rhsStash'] {
```

Comment:
I see code above that appears to handle plugins, but just confirming explicitly that this still works with plugin content in the RHS?